src/_includes/macros/video-player.html

```diff
@@ -0,0 +1,80 @@
+{# ==========================================================================
+
+   video-player.render()
+
+   ==========================================================================
+
+   Description:
+
+   Builds Video markup when given:
+
+   options             : An object used to customize the Video markup.
+
+   options.video.player: A String used to denote the type of video player.
+
+   options.video.id    : A String used to denote the id of the video.
+
+   options.video.url   : A String used to denote the url of the video.
+
+   options.image.url   : A String used to denote the url of the image.
+
+   options.button_pos  : A String used to denote the position of the video
+                         play button.
+
+                         Possible Values: 'center, bottomRight'
```

Comment:
fixed in https://github.com/cfpb/cfgov-refresh/commit/247f9ed0a517e05adc872503553df83bba694d2b